sandbox/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,6 @@
 <resources>
     <string name="app_name">Appyx sandbox</string>
+    <string name="rib_root_title">Black area belongs to 1.0, container below to 2.0</string>
```

Comment:
I think you could just add `HardcodedText` to Lint ignore list for sandbox/app projects.

Reply:
done